src/main/java/org/broadinstitute/consent/http/db/LibraryCardDAO.java

```diff
@@ -133,4 +135,19 @@ INSERT INTO lc_daa (lc_id, daa_id)
       AND daa_id = :daaId
       """)
   void deleteLibraryCardDaaRelation(@Bind("lcId") Integer lcId, @Bind("daaId") Integer daaId);
+
+  /**
+   * Finds library cards by user emails.
+   * @param emails A list of LOWER cased email addresses
+   * @return List of LibraryCard objects associated with the provided emails.
+   */
+  @RegisterBeanMapper(value = LibraryCard.class)
+  @UseRowReducer(LibraryCardReducer.class)
+  @SqlQuery("""
+      SELECT *
+      FROM library_card
+      WHERE LOWER(user_email) in (<emails>)
```

Comment:
```suggestion
      WHERE lower(user_email) = any(array(SELECT lower(unnest(<emails>))));
```

Ugly, but does the entire lowercase in the query instead of half in the query and half in the code.